app/views/hyrax/admin/admin_sets/edit.html.erb

```diff
@@ -1,3 +1,6 @@
+<%# Needed for bootstrap tabs use on this page (https://github.com/projecthydra-labs/hyrax/issues/254) %>
```

Comment:
Can you be even more descriptive?  Why are we turning off turbolinks?  There is nothing in this ticket about turbolinks: https://github.com/projecthydra-labs/hyrax/issues/254